gazebo_ros/launch/gzserver.launch.py

```diff
@@ -55,12 +55,10 @@ def generate_launch_description():
         # _plugin_command('force_system'), ' ',
         _arg_command('profile'), ' ', LaunchConfiguration('profile'), ' ',
         # convenience parameter for params file
-        '--ros-args',
-        PythonExpression([
-            '"--params-file', '" if "" != "',
-            LaunchConfiguration("params_file"), '" else ""']),
-        LaunchConfiguration("params_file"),
-        '--',
+        _arg_command('--ros-args', condition=LaunchConfiguration('params_file')), ' ',
+        _arg_command('--params-file', condition=LaunchConfiguration('params_file')),
+        ' ', LaunchConfiguration('params_file'), ' ',
+        _arg_command('--', condition=LaunchConfiguration('params_file')), ' ',
```

Comment:
`_arg_command` is already prepending `--`. Also, the extra spaces seem to interfere with ROS argument parsing. I was able to get this to work with the following diff:

```suggestion
        _arg_command('ros-args', condition=LaunchConfiguration('params_file')),
        _arg_command('params-file', condition=LaunchConfiguration('params_file')),
        LaunchConfiguration('params_file'),
        _arg_command('', condition=LaunchConfiguration('params_file')), ' ',
```

Reply:
Using the `_arg_command` wrapper is a bit cryptic IMO, versus just using `PythonExpression` directly here. But I'm fine approving this either way.

Reply:
Using the `_arg_command` wrapper is a bit cryptic IMO, versus just using `PythonExpression` directly here. But I'm fine approving this either way.

Reply:
I prefer the wrapper since it is more consistent with the rest of the file